src/app/feed/feed-advanced-search/feed-advanced-search.component.html

```diff
@@ -0,0 +1,38 @@
+	<div class="tabslist">
+		<ul>
+			<li [class.active_view]="show('all')" class="tabs" (click)="getAllResults()">All</li>
+			<li [class.active_view]="show('image')" class="tabs" (click)="getImageResults()">Images</li>
+			<li [class.active_view]="show('video')" class="tabs" (click)="getVideoResults()">Videos</li>
+			<div class="gap"></div>
+			<li class="tabs" (click)="showTools = (!showTools)">Settings</li>
```

Comment:
how about naming this as `filters` or `refine` or something fancy 'sift it!!' instead of `setting`